app/javascript/data.js

```diff
@@ -0,0 +1,207 @@
+import createDebug from 'debug';
+import { onLoad } from './util';
+
+const debug = createDebug('ms:data');
+
+window.store = {};
+window.results = [];
```

Comment:
It’s generally best not to store data in global variables. Can you store it in a `let` (or preferably `const`) variable instead?